code/game/objects/items/credit_holochip.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/holochip
```

Comment:
You're going about this wrong. Just make this a subtype of space cash.

Reply:
1) Stack behaviour complicates calculations, see the bits about the bank machine syphoning
2) Since you can make/unmake these at will there is no in-universe need for separate denominations
3) Stacking and merging are intentionally removed for now since you can slap them into an ID and take a custom amount out, but re-adding it won't be complicated if it happens.